cmd/iam.go

```diff
@@ -264,6 +264,7 @@ func loadUser(objectAPI ObjectLayer, user string, isSTS bool,
 		return nil
 	}
 
+	u.Credentials.AccessKey = user
```

Comment:
Don't think this is needed

Reply:
This is needed for users who may be on the released (bad) migration. Their users will start working if they restart with this change once released.

Reply:
Then should we add a comment as to why this line is added?

Reply:
```suggestion
        // In some cases access key may not be set, so we set it explicitly.
	u.Credentials.AccessKey = user
```